06-llm-pr-review/backend/main.py

```
@@ -0,0 +1,99 @@
+    CORSMiddleware,
+    allow_origins=origins,
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+@app.post("/products/", response_model=ProductDTO)
+async def create_product(
+    payload: ProductCreate,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)